packages/components/src/widgets/LambdaWidget/FunctionChart/index.tsx

```diff
@@ -63,14 +65,23 @@ export const FunctionChart: FC<FunctionChartProps> = ({
   const signatures = fn.signatures();
   const domain = signatures[0][0]?.domain;
 
-  const min = domain?.min ?? settings.functionChartSettings.start;
-  const max = domain?.max ?? settings.functionChartSettings.stop;
+  const min: number = domain?.min ?? settings.functionChartSettings.start;
+  const max: number = domain?.max ?? settings.functionChartSettings.stop;
```

Comment:
This means that plots with `Scale.date()` without min and max would have the default range from 1970 midnight to 10 milliseconds past the midnight.

https://squiggle-website-git-datescale-quantified-uncertainty.vercel.app/playground?v=dev#code=eNqrVkpJTUsszSlxzk9JVbJSCsjJL9HLK81NLcpMdsvTqI7JU1BIy7NSqK6pqKmo1QFxK4KTE3NSrRTAlF5KYkmqhmZMXq2mUi0AJSMarw%3D%3D

Should we add separate `startDate` and `stopDate` settings?

Reply:
Gr... that's a good point.

Reply:
An easier way of doing this, that I bet users would be fine with now, is that in this case - we just map start and stop to years from today, or something like that.